examples/multiple_secrets/k8s/Deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       serviceAccountName: example
       containers:
       - name: main
-        image: vsecm/example-multiple-secrets:0.26.2
+        image: vsecm/example-multiple-secrets:latest
```

Comment:
Pinning version was causing issues in quickstart tutorials.

This would be easier to manage.

Reply:
We are setting `:latest` only for things under the `examples` folder.

The rest of the apps still have whatever the future version under development is.